src/main/app/claims/models/claim.ts

```diff
@@ -92,6 +92,20 @@ export class Claim {
     return this.respondedAt.clone().add('33', 'days')
   }
 
+  get respondToDefendantPaidInFullDeadline (): Moment {
+    if (!this.respondedAt) {
+      return undefined
+    }
+    return this.respondedAt.clone().add('28', 'days')
+  }
+
+  get respondToMediationDeadline (): Moment {
+    if (!this.respondedAt) {
+      return undefined
+    }
+    return this.respondedAt.clone().add('5', 'days')
```

Comment:
the same as above

Reply:
This is a new deadline respondToMediationDeadline is only 5 days